run.py

```diff
@@ -0,0 +1,52 @@
+import json, requests, os
+from flask import Flask, Response, abort, request
+from os.path import join, dirname
+from watson_developer_cloud import VisualRecognitionV3
+
+app = Flask(__name__, static_url_path='')
+app.config['PROPAGATE_EXCEPTIONS'] = True
+port = os.getenv('VCAP_APP_PORT', '5000')
+
+# Global variables for credentials
+url = ''
+apikey = ''
+
+def get_vcap_credentials(vcap_env, service):
```

Comment:
i *think* you could just do:

```
from watson_developer_cloud import watson_service
watson_service.load_from_vcap_services(service_name)
```

https://github.com/watson-developer-cloud/python-sdk/blob/master/watson_developer_cloud/watson_service.py#L38-L45

instead of building your own